packages/integration-tests/src/__tests__/integration/mariadb/__database.ts

```diff
@@ -25,6 +25,7 @@ export const database = {
     CREATE DATABASE ${ctx.id};
     USE ${ctx.id};`
     await db.query(sqlUp + sqlScenario)
+    await db.end()
```

Comment:
Why is this needed?

Reply:
I'll be honest, in retrospect it doesn't seem useful

Reply:
`git blame` tells me that @Jolg42 added it

Reply:
Yes it's a nice improvement we already did that for SQL Server only I think, found out when we had the exhaustion of connections on PostrgeSQL